Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -46,12 +46,13 @@ public override string ToString()
         }
     }
     
+    [IonDoNotAnnotateType(ExcludeDescendants = true)]
     public class Motorcycle : Vehicle
     {
         public string Brand { get; init; }
 
         [IonField]
-        private string color;
+        public string color;
```

Comment:
Why must this be public?

Reply:
So the test can call `Assert.IsNull(deserialized.color);`